client/src/components/Canvas.tsx

```diff
@@ -1,24 +1,96 @@
-import React from 'react'
+import React, { useEffect, useRef, useState, MouseEvent } from 'react'
 import { TransformWrapper, TransformComponent } from 'react-zoom-pan-pinch/dist'
 import { makeStyles } from '@material-ui/core'
 
 import map from '../maps/rl_pitch.png'
 
-const useStyles = makeStyles({
-  image: {
-    width: '100vw'
+let ctx: CanvasRenderingContext2D | null
+
+export default function Canvas() {
+  const _board = useRef<HTMLCanvasElement>(null)
+  const [isPainting, setIsPainting] = useState(false)
+  const [prevPos, setPrevPos] = useState<IClient>({ clientX: 0, clientY: 0 })
+  const [line, setLine] = useState<ILine[]>([])
+  const [userStrokeStyle, setUserStrokeStyle] = useState('#EE92C2')
+
+  useEffect(() => {
+    if (_board.current) {
+      console.log(_board.current)
+      _board.current.width = 1000
```

Comment:
is there a benefit to initialize the width and height here, rather than in the DOM itself?

Reply:
I believe so, in case the user resizes the window it makes for easy adjusting

Reply:
👍